capstone/src/test/javascript/graphsTest.js

```diff
@@ -0,0 +1,15 @@
+describe('Data Table Construction', function() {
+  it('Will add correct columns/rows to DataTable from JSON', function() {
+    const expectedData;
```

Comment:
This is marked as const and then its value is set later. 
Same comment for the other fields.